docs/docs/dev_docs/contracts/deploying.md

```diff
@@ -1 +1,60 @@
-See sandbox section?
\ No newline at end of file
+# Deploying contracts
+
+Once you have [compiled](./compiling.md) your contracts you can proceed to deploying them using `aztec-cli`.
+
+## Prerequisites
+- aztec-cli installed (go to [CLI main section](./main.md) for installation instructions)
+- contract artifacts ready (go to [Compiling contracts section](../contracts/compiling.md) for instructions on how to compile contracts)
+- aztec-sandbox running (go to [Sandbox section](../sandbox/main.md) for instructions on how to install and run the sandbox)
+
+## Deploy
+
+To deploy the contracts we use the `deploy` command form `aztec-cli`:
+
+```bash
+aztec-cli deploy /path/to/contract/abi.json
+```
+
+### Arguments
+This command takes 1 mandatory positional argument which is the path to the contract ABI file in a JSON format (e.g. `build/contracts/PrivateToken.json`). Alternatively you can pass here a name of a contract ABI as exported by `@aztec/noir-contracts` (run `aztec-cli example-contracts` to get the list).
+
+The command also takes the following optional arguments:
+- `-args <constructorArgs...>` (default: `[]`): Arguments to pass to the contract constructor.
+- `--rpc-url <string>` (default: `http://localhost:8080`): URL of the Aztec node to connect to.
+- `--public-key <string>` (default: `undefined`): Optional encryption public key for this contract.
+Set this only if this contract is expected to receive private notes (in such a case the public key is used during the note encryption).
+> **NOTE**: Contracts which are expected to receive notes are called account contracts.
+- `--salt <string>` (default: random value): Hexadecimal string used when computing the contract address of the contract being deployed.
+By default is set to a random value.
+Set it, if you need a deterministic contract address (same functionality as Ethereum's `CREATE2` opcode).
+
+To give you a more complete example we will deploy the `PrivateToken` contract whose artifacts are included in the `@aztec/noir-contracts` package.
+
+### Deploying private token contract
+The contract has `initial_supply` and `owner` as constructor arguments.
+Because the contract sends a note to the owner specified inside the constructor, we first need to register the owner as a recipient inside the Aztec RPC with the following command:
```

Comment:
Let's clarify that this is not needed if the recipient is one of the `accounts`

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/1633/commits/9dea419a43ca4bc74ddd8fecb7bd462c7cb6eca6